KifuSF/Report.swift

```diff
@@ -21,18 +21,28 @@ enum FlaggedContentType: Int {
     case flaggedPhoneNumber = 100
     case flaggedCommunication
     
-    func getDescription() -> String {
+    var description: String {
         switch self {
         case .flaggedImage:
             return "Inapropiate Image"
         case .flaggedPickupLocation:
             return "Wrong pickup location"
-        default:
-            return "poof"
+        case .flaggedNotes:
+            return "Inapropiate/Misleading Note"
+        case .flaggedVerificationImage:
+            return "Inapropiate/Misleading Verification Image"
+        case .flaggedPhoneNumber:
+            return "Incorrect Phone Number"
+        case .flaggedCommunication:
+            return "Inapropiate/Unresponsive Communication"
```

Comment:
Adjust the string values to what you feel is best, please

Reply:
done!